libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/idp/SPManager.kt

```diff
@@ -171,23 +183,44 @@ internal class SPManager(
         send(context, message, idpAppPackageName)
     }
 
+    fun sendLoginRequest(context: Context, message: SPToIDPRequest) {
+        if (context is Activity) {
```

Comment:
SP initiated login flow:
- start IDP activity for auth code
IDP initiated login flow:
- send broadcast and have the IDP app start the IDP activity for auth code